.github/workflows/publish-release.yml

```diff
@@ -0,0 +1,125 @@
+name: Publish Release
+run-name: ${{ github.event.pull_request.head.ref }}
+
+on:
+  pull_request_target:
+    branches:
+      - master
+    types:
+      - closed
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  publish:
+    if: github.event.pull_request.merged && startsWith(github.event.pull_request.head.ref, 'release/')
+    runs-on: ubuntu-latest
+    outputs:
+      tag: ${{ steps.tag.outputs.version }}
+    steps:
+    - name: Check out code
+      uses: actions/checkout@v3
+      with:
+        ref: ${{ github.event.pull_request.merge_commit_sha }}
+        fetch-depth: 0
+        fetch-tags: true
+
+    - name: Tag release
+      run: |
+        # Set up github-actions[bot] user
+        git config --local user.name "github-actions[bot]"
+        git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
```

Comment:
Where did this magic bot name come from?

Reply:
It is the "official" email address of the github actions bot. GitHub does not officially mention it in the documentation but you find it used everywhere. I guess people just looked at the commit history and figured out that's the email and user id github is using for it's bot. Even the official google release action is using this bot internally: https://github.com/google-github-actions/release-please-action/blob/main/.github/workflows/release-please.yaml#L76